app/models/ncr/work_order.rb

```diff
@@ -103,7 +103,9 @@ def setup_approvals_and_observers(approving_official_email)
         self.proposal.update(status: 'approved')
       else
         approvers = emails.map{|e| User.for_email(e)}
+        removed_approvers_to_notify = self.proposal.approvals.non_pending.map(&:user) - approvers
```

Comment:
Not a blocker, but would probably be useful to have a `scope :non_pending_approvers` on the `Approval` model.